app/src/main/java/org/schabi/newpipe/local/playlist/LocalPlaylistFragment.java

```diff
@@ -70,6 +70,8 @@
 import io.reactivex.rxjava3.subjects.PublishSubject;
 
 public class LocalPlaylistFragment extends BaseLocalListFragment<List<PlaylistStreamEntry>, Void> {
+    public static final long DEFAULT_THUMBNAIL_ID = -1;
+    public static final long NO_THUMBNAIL_ID = -2;
```

Comment:
Since this value is only used internally in `LocalPlaylistManager`, remove it from here and move it there. I would also call it `THUMBNAIL_ID_LEAVE_UNCHANGED`.